.github/pull_request_template.md

```
@@ -1,35 +1,15 @@
-## PR Checklist
+## Summary of Changes
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)